.changeset/angry-beds-juggle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@premieroctet/next-admin-cli": major
```

Comment:
Since it's the first publication of this package, we can start with a patch or minor at the most
```suggestion
"@premieroctet/next-admin-cli": patch
```

Reply:
This is a major so that it becomes 1.0.0 when changeset publishes it